### PR TITLE
Mirror Safari Desktop -> Safari iOS for css/*

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
This PR mirrors the values of Safari Desktop onto Safari iOS when Safari iOS is `true` or `null`, to help reduce data inconsistencies and reach towards the 2019 Key Result.  Safari versions that were "6" or lower were altered to state the correct iOS editions.

The script that performed the majority of changes can be found here: https://github.com/vinyldarkscratch/bcd-toolkit/blob/master/fix.py

### CODE REVIEW

File: css/properties/cursor.json [hunk 1]
```diff
@@ -226,7 +226,7 @@
                 "version_added": "3"
               },
               "safari_ios": {
-                "version_added": null
+                "version_added": "4.2"
```
Comment (Elchi3) [inline css/properties/cursor.json]:
```suggestion
                "version_added": "1"
```

File: css/properties/hyphens.json [hunk 2]
```diff
@@ -413,7 +413,7 @@
                 "version_added": "5.1"
               },
               "safari_ios": {
-                "version_added": null
+                "version_added": false
```
Comment (Elchi3) [inline css/properties/hyphens.json]:
```suggestion
                "version_added": "5"
```

File: css/properties/hyphens.json [hunk 3]
```diff
@@ -560,7 +560,7 @@
                 "version_added": false
               },
               "safari_ios": {
-                "version_added": null
+                "version_added": "5"
```
Comment (Elchi3) [inline css/properties/hyphens.json]:
```suggestion
                "version_added": false
```

File: css/properties/hyphens.json [hunk 4]
```diff
@@ -1376,7 +1376,7 @@
                 "version_added": "5.1"
               },
               "safari_ios": {
-                "version_added": null
+                "version_added": false
```
Comment (Elchi3) [inline css/properties/hyphens.json]:
```suggestion
                "version_added": "5"
```

File: css/properties/min-width.json [hunk 5]
```diff
@@ -252,9 +252,15 @@
...
               ],
-              "safari_ios": {
-                "version_added": null
-              },
+              "safari_ios": [
+                {
+                  "version_added": "11"
+                },
+                {
+                  "alternative_name": "min-intrinsic",
```
Comment (Elchi3) [inline css/properties/min-width.json]: This is just intrinsic on safari. Why would it be min-intrinsic on iOS?

```suggestion
                  "alternative_name": "intrinsic",
```

File: css/properties/width.json [hunk 6]
```diff
@@ -366,7 +366,7 @@
                 }
               ],
               "safari_ios": {
-                "version_added": null
+                "version_added": false
```
Comment (Elchi3) [inline css/properties/width.json]: Safari has an array compat statement here. Shouldn't that be copied over?

File: css/properties/width.json [hunk 7]
```diff
@@ -438,7 +438,7 @@
...
```
Comment (Elchi3) [inline css/properties/width.json]: Same here. Safari isn't false.

**Comment by vinyldarkscratch** [comment]
It probably was the result of a poor rebase, so sorry for all of the issues!  I'll resolve them ASAP.  :wink:

**Comment by Elchi3** [comment]
Thanks for the quick fixes! Now the changes look good to me. I ran ` npm run traverse safari_ios css` on this branch and there are still 141 results. Is this only a partial mirroring?

**Comment by vinyldarkscratch** [comment]
My best guess is that there was some new Safari values added since I ran the script.  There were also some arrays which apparently weren't changed during the script, but moreso some properties didn't even have a `safari_ios` statements.  I'm pushing some commits to complete this PR!

File: css/properties/cursor.json [hunk 8]
```diff
@@ -841,7 +841,7 @@
                 "version_added": "5"
               },
               "safari_ios": {
-                "version_added": null
+                "version_added": "1"
```
Comment (Elchi3) [inline css/properties/cursor.json]:
```suggestion
                "version_added": "4.2"
```

**Comment by Elchi3** [comment]
I've filed https://github.com/mdn/browser-compat-data/issues/4719 on that matter.
Since I've manually vetted this mirroring, I'm going to merge this PR.

**Pull request merged**